### DISCUSSION TITLE
发送包大小能否模拟超过MTU的情况

### DISCUSSION THREAD

**Discussion by Pars1f4l** [baidu/dperf#443] [Q&A]
这边测试下来client端发送的包大小不能超过当前设置的MTU大小(默认应该是1500)，同样payload_size也有相应限制，这边启用jumbo发送巨型帧的话能够实现发送大负载的包，这边的问题是：
1. 发送超过MTU大小的数据包 当前的处理逻辑是什么，dperf端是无法按照MTU进行分包发送吗？
2. 如果要模拟超过MTU大小的数据包 dperf有没有解决办法，或者有没有别的更优的方法能够模拟？

**Comment by pengjianzhang** [comment]
你是要测试IP分片吗？dperf不支持分片。

**Reply by Pars1f4l** [reply]
ok。那再请问下原数据包是2倍的MTU，分包后发送的情况能够用dperf发1MTU的包模拟吗

**Reply by pengjianzhang** [reply]
我没有看懂你的问题；请把你问题的背景说明一下，测试什么设备，为什么要考虑分片等

**Reply by Pars1f4l** [reply]
这边想测试dpvs的性能指标，是不是考虑分片情况没有必要，因为分片和重组是client和server负责的，dpvs只负责转发和新建连接

**Reply by pengjianzhang** [reply]
没有必要

**Reply by Pars1f4l** [reply]
好的，感谢回复

**Reply by pengjianzhang** [reply]
请向周围小伙伴宣传一下dperf 谢谢
